spec/01-unit/27-queue_spec.lua

```diff
@@ -342,7 +342,7 @@ describe("plugin queue", function()
     enqueue("another value")
     wait_until_queue_done("retry-give-up")
     assert.match_re(log_messages, 'WARN .* handler could not process entries: FAIL FAIL FAIL')
-    assert.match_re(log_messages, 'ERR .*1 queue entries were lost')
+    assert.match_re(log_messages, 'ERR .* 7 queue entries were lost')
```

Comment:
Why do we need to change this to 7 to make the ci pass? I think the queue logic does not change.